Source/Shared/Cache/SpecializedCache.swift

```diff
@@ -13,25 +13,25 @@ public final class SpecializedCache<T: Cachable>: BasicHybridCache {
    - Parameter expiry: Expiration date for the cached object
    - Parameter completion: Completion closure to be called when the task is done
    */
-  public func add(_ key: String, object: T, expiry: Expiry? = nil, completion: (() -> Void)? = nil) {
-    super.add(object, forKey: key, expiry: expiry, completion: completion)
+  public func add(_ key: String, object: T, expiry: Expiry? = nil, completion: Completion? = nil) {
+    return super.add(object, forKey: key, expiry: expiry, completion: completion)
```

Comment:
Yes, but `SpecializedCache` is generic and superclass is not, so it does the trick.